src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetReimageMethod.cs

```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetReimageMethod(object[] invokeMethodI
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.Reimage(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.Reimage(resourceGroupName, vmScaleSetName, instanceIds);
```

Comment:
breaking change?

Reply:
It looks liek the below actually writes the value, so, what is this method change for?